### ISSUE TITLE
Missing dependency

### ISSUE THREAD

**Issue by gf3** [romeovs/react-a11y#2]
Getting the following error when attempting to use your version of `react-a11y` in my project:

```
WARNING in ./~/react-a11y/lib/util/test-rules.js
Critical dependencies:
28:23-69 the request of a dependency is an expression
 @ ./~/react-a11y/lib/util/test-rules.js 28:23-69

ERROR in ./~/react-a11y/lib/util/test-rules.js
Module not found: Error: Cannot resolve module 'chai' in /Users/gianni.chiappetta/Projects/uniiverse/juno/node_modules/react-a11y/lib/util
 @ ./~/react-a11y/lib/util/test-rules.js 99:12-27
webpack: bundle is now VALID.
```

**Comment by gf3** [comment]
After installing the dependency i get the following warning:

```
WARNING in ./~/react-a11y/lib/util/test-rules.js
Critical dependencies:
28:23-69 the request of a dependency is an expression
 @ ./~/react-a11y/lib/util/test-rules.js 28:23-69
```

Which is [here](https://github.com/gf3/react-a11y/blob/publish-lib/lib/util/test-rules.js#L28).

**Comment by romeovs** [comment]
Hi @gf3 
Thanks for looking into it! 
I probably needed to mention that to use `test-rules.js` you need to install `chai` yourself. It is in `react-a11y`s `devDependencies` and so it's not installed automatically.

The second error is harder to fix. I need to look at the web pack config to make sure the require context is generated somehow. This should not be a problem when you are only doing server-side tests though.

**Comment by gf3** [comment]
hi @romeovs i actually can't get the package to output any a11y warnings at all (no errors either). do i need to manually include all the rules i'd like to use?

**Comment by romeovs** [comment]
By default all the rules are turned `"off"` so you'd need to turn them on:

``` js
a11y(React, ReactDOM, {
    rules: {
      'img-uses-alt': 'warn'
    , 'redundant-alt': 'warn'
    // all the rules!
    }
  })
```

**Comment by gf3** [comment]
ahh thanks! i would consider adding that to the README

**Comment by romeovs** [comment]
I will!

**Comment by romeovs** [comment]
This should be fixed with [9e43ec32](https://github.com/romeovs/react-a11y/commit/9e43ec32185ed528a047131dbf9ba6f6d9573e23). By including `./src/util/test-rules` in the source tree, the error would occur even when you didn't really need `test-rules`. This is fixed now!
